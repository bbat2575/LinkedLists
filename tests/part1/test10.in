TYPE 0
NEW 4
1
2.2
E
Four
NEW 1
2e4
TYPE +1
TYPE -1
TYPE +0
TYPE -0
TYPE +
TYPE -
TYPE 2
TYPE o
TYPE O
TYPE ZERO
TYPE .
TYPE 0.
TYPE .0
TYPE 0.0
TYPE 0e0
TYPE 2e-6
TYPE 0 0
TYPE  0
TYPE   0
TYPE
TYPE  
  TYPE
0TYPE
0 TYPE
TYPE0
 TYPE 0
  TYPE 0
TYP E 0
TY PE 0
T YPE 0
type 0
Type 0
TypE 0
TyPE 0
YTPE 0
