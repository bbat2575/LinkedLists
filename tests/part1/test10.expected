
Type HELP for a list of options
Press <CTRL + D> to Quit

> INVALID COMMAND: TYPE

> List 0: 1 -> 2.20 -> E -> Four

> List 1: 20000.00

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: TYPE

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
